src/bootstrap/bootstrap.py

```diff
@@ -952,14 +952,18 @@ def program_config(self, program, stage0=True):
 
     @staticmethod
     def get_string(line):
-        """Return the value between double quotes
+        """Return the value between double quotes inside the string with single quotes or double quotes
```

Comment:
I'm not sure I follow this change -- I think the "string with single quotes or double quotes" is describing just Python strings, not something specific to this API? In that case, this is akin to adding a type annotation to `line` of `: str`, but that is pretty obvious I suspect regardless based on the examples/tests, so I'm not sure that saying it in words is worthwhile.